pkg/chaining/transition.go

```diff
@@ -0,0 +1,44 @@
+package chaining
+
+import (
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/configuration"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/service"
+	"github.com/TurnsCoffeeIntoScripts/jira-api-issue-resource/pkg/status"
+)
+
+func PerformForceOpen(params *configuration.JiraAPIResourceParameters) bool {
+	if err := fetchTransitionsIfMissing(params); err != nil {
+		return false
+	}
+
+	srvDoTransition := &status.ServiceDoTransition{}
+	if err := service.Execute(srvDoTransition, *params, false); err != nil {
+		return false
+	}
+
+	return true
+}
+
+func PerformClose(params *configuration.JiraAPIResourceParameters) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported function PerformClose should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=5024349)